src/pilot/dracclient_raid.patch

```diff
@@ -1,5 +1,5 @@
---- /usr/lib/python2.7/site-packages/dracclient/resources/raid.py	2019-05-09 15:37:11.450522239 +0000
-+++ raid.py	2019-05-09 15:37:04.219391200 +0000
+--- /usr/lib/python2.7/site-packages/dracclient/resources/raid.py	2019-07-11 15:22:53.819829111 +0000
```

Comment:
I believe you can strip the date and time stamps from the file 
i.e.   delete  "2019-05-09 15:37:04.219391200 +0000" & "2019-07-11 15:22:53.819829111 +0000" and the patch file will still work.   Not a concern here, just an FYI.

Reply:
k - I just do a "diff -Naur ... > blah.patch"